test/fixture/terraform_15_1.0_filter/aws_azure/expected/main.terratag.tf

```diff
@@ -0,0 +1,47 @@
+terraform {
+  required_providers {
+    azurerm = {
+      source = "hashicorp/azurerm"
+    }
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 2.0"
+    }
+  }
+}
+
+provider "azurerm" {
+  features {}
+}
+
+provider "aws" {
+  region = "us-east-1"
+}
+
+resource "azurerm_resource_group" "example" {
```

Comment:
Could be helpful to give those resources some meaningful name per their expected status like `should_have_tags` or `should_not_have_tags`

Reply:
@yaronya - great call, added